docs/features/kubernetes/installation.md

```diff
@@ -72,6 +72,7 @@ export default async function createPlugin(
     logger: env.logger,
     config: env.config,
     catalogApi,
+    permissions,
```

Comment:
I don't think the variable `permissions` is in this scope
```suggestion
    permissions: env.permissions,
```

Reply:
My mistake. Good catch added.